app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    cmd, origin, dupe = command.split()
```

Comment:
Code Efficiency: It's a good practice to check if the `command` string can be split into exactly three parts before attempting to do so. This will prevent a `ValueError` if the `command` string does not contain exactly two spaces.